content/rdi/monitoring-guide.md

```diff
@@ -5,103 +5,68 @@ description: Monitor RDI Engine and data processing jobs
+RDI allows collecting and exporting its metrics to [Prometheus](https://prometheus.io/) and visualizing them in [Grafana](https://grafana.com/). Operator can start the built-in exporter using the [`redis-di monitor`]({{<relref "/rdi/reference/cli/redis-di-monitor">}}) command. The diagram below describes this flow and components involved.
 
-![Metrics architecture](/images/rdi/monitoring-architecture.png)
+![Metrics architecture](/images/rdi/monitoring-diagram.png)
 
 > Note: The host names and ports above are examples only and can be changed as needed.
 
-## Install and configure the Exporter
+### Test RDI metrics exporter
 
-OSS Metrics Exporter is available as a [pre-built docker container](https://hub.docker.com/r/oliver006/redis_exporter) so that you integrate it into container-based or Kubernetes environments. Alternatively, you can be [build and install](https://hub.docker.com/r/oliver006/redis_exporter) it as a binary to any compute node that has access to an RDI database that needs to be monitored.
+Start the RDI metrics exporter using the command below:
 
-To connect OSS Metrics Exporter to the RDI database, provide the following information by using the command-line options or environment variables:
-
-| Variable Name         | Description                                                                                      | Example                  |
-| --------------------- | ------------------------------------------------------------------------------------------------ | ------------------------ |
-| REDIS_ADDR            | RDI database host/port                                                                           | redis://localhost:12001  |
-| REDIS_USER            | RDI database user (optional, if `default` is used)                                               | RedisUser                |
-| REDIS_PASSWORD        | RDI database password                                                                            | Redis123                 |
-| REDIS_EXPORTER_SCRIPT | Lua script that triggers the Metrics Collector [(see below)](#lua-script-for-metrics-collection) | /scripts/rdi_metrics.lua |
-
-### Lua script for metrics collection
-
-Create the following [Lua script](https://redis.io/docs/manual/programmability/eval-intro/) and make it available for the OSS Metrics Exporter by using the `REDIS_EXPORTER_SCRIPT` environment variable:
-
-```lua
-return (redis.call('RG.TRIGGER', 'GetMetrics', '*'))[1]
+```bash
+redis-di monitor
 ```
 
-### Test OSS Metrics Exporter
+> Note: The default port for the exporter is `9121`. If you need to change it, use the `--exporter-port` option. The default metrics collection interval is 5 seconds. If you need to change it, use the `--collect-interval` option.
```

Comment:
Also point to the reference of the monitor command like done beforehand in a different section.

Reply:
I'd say it is redundant here because it is a not to the command explanation containing the reference.

Reply:
@redima please link the flags to the command reference